scripts/install-pedatious.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+# exit on any error
+
+#STILL UNDER CONSTRUCTION
+set -Ee
+
+PADATIOUS_DIR=padatious
+CORES=$(nproc)
+PADATIOUS_VERSION=0.1
+
+# for ubuntu precise in travis, that does not provide pkg-config:
+pkg-config --exists icu-i18n || export CFLAGS="$CFLAGS -I/usr/include/x86_64-linux-gnu"
+pkg-config --exists icu-i18n || export LDFLAGS="$LDFLAGS -licui18n -licuuc -licudata"
+
+
+#install pre-reqs for building
+sudo apt-get install python3 ninja-build build-essential \
+    && pip3 install --user meson
+
+# download and install pedatious
+if [ ! -d ${PADATIOUS_DIR} ]; then
+    #TODO: add versioning later
+    git clone https://github.com/MycroftAI/pedatious.git
```

Comment:
Change the `e` to `a` and, perhaps the url should be `padatious-mycroft` since that's actually [the component](https://github.com/MatthewScholefield/padatious-mycroft) that will run alongside Mycroft; the other is just a plain library.